pilot/pkg/config/kube/ingress/controller.go

```diff
@@ -202,9 +190,6 @@ func (c *controller) List(typ, namespace string) ([]model.Config, error) {
 		case model.Gateway.Type:
 			gateways, _ := ConvertIngressV1alpha3(*ingress, c.domainSuffix)
```

Comment:
rename `ConvertIngressV1alpha3` as well?

Reply:
guess this will depend on the outcome of the above decision.